PromoExchange/batman
# Adjust delivery date

https://trello.com/c/bS1xRWI2
### Testplan
1. Open company store
2. Selected != UPS Ground shipping
3. Ensure shipping date includes production time
4. Press purchase
5. Return to main screen, select quantity
6. Ensure selected shipping option is reset to UPS Ground
